base_layer/wallet_ffi/wallet.h

```diff
@@ -597,6 +600,21 @@ unsigned long long balance_get_pending_incoming(struct TariBalance *balance, int
 // Gets the available balance from a TariBalance
 unsigned long long balance_get_pending_outgoing(struct TariBalance *balance, int *error_out);
 
+// Gets the public_key from a TariContactsLivenessData
+struct TariPublicKey *liveness_data_get_public_key(struct TariContactsLivenessData *liveness_data, int *error_out);
+
+// Gets the latency from a TariContactsLivenessData
+int liveness_data_get_latency(struct TariContactsLivenessData *liveness_data, int *error_out);
+
+// Gets the last_seen from a TariContactsLivenessData
+char *liveness_data_get_last_seen(struct TariContactsLivenessData *liveness_data, int *error_out);
+
+// Gets the message_type from a TariContactsLivenessData
```

Comment:
What are the enum options encoded by this int? Shoul be in the docstring comment